authority/ssh.go

```diff
@@ -161,6 +162,8 @@ func (a *Authority) SignSSH(ctx context.Context, key ssh.PublicKey, opts provisi
 	// Set backdate with the configured value
 	opts.Backdate = a.config.AuthorityConfig.Backdate.Duration
 
+	opts.WebhookClient = a.webhookClient
```

Comment:
we don't need this if we pass the client to the authorizer function, it might look cleaner.